hera/GameMap.tsx

```diff
@@ -62,6 +62,7 @@ import { resetBehavior, setBaseClass } from './behavior/Behavior.tsx';
 import MenuBehavior from './behavior/Menu.tsx';
 import NullBehavior from './behavior/NullBehavior.tsx';
 import Cursor from './Cursor.tsx';
+import MapEditorExtraCursors from './editor/MapEditorExtraCursors.tsx';
```

Comment:
What about calling it `MapEditorMirrorCursors.tsx`?

Reply:
👍  430a4f7e68cb1220225cd4d76ce9adceaa61f97d